docs/reference/glossary.md

```diff
@@ -0,0 +1,161 @@
+
+## Block height
+
+The _height_ of a [block](#block) corresponds to the number of [epochs](#epoch) elapsed before the block was added to the blockchain. The height of the Filecoin [blockchain](#blockchain) is defined to be the maximum height of any block in the blockchain.
+
+## Capacity commitment
+
+If a storage miner doesn't find any available deal proposals appealing, they can alternatively make a _capacity commitment_, filling a [sector](#sector) with arbitrary data, rather than with client data. Maintaining this sector allows the storage miner to provably demonstrate that they are reserving space on behalf of the network.
+
+## Collateral
+
+In order to enter into a [storage deal](#deal), a [storage miner](#storage-miner) is required to provide filecoin as _collateral_, to be paid out as compensation to a client in the event that the miner fails to uphold their storage commitment.
+
+## Deal
+
+Two participants in the Filecoin network can enter into a _deal_ in which one party contracts the services of the other. The Filecoin specification currently details _storage deals_ (in which one party agrees to store data for the other for a specified length of time) and _retrieval deals_ (in which one party agrees to transmit specified data to the other).
+
+## Election
+
+Every [epoch](#epoch), a small subset of Filecoin [storage miners](#storage-miner) are _elected_ to mine a new [block](#block) for the Filecoin blockchain. A miner's probability of being elected is roughly proportional to the share of the Filecoin network's total storage capacity that they contribute.
+
+## Epoch
+
+Time in the Filecoin blockchain is discretized into _epochs_ that are currently twenty-five seconds in length. Each epoch, a subset of storage miners are elected to each add a new block to the Filecoin blockchain via [Winning Proof-of-Spacetime](#winning-proof-of-spacetime-winningpost).
+
+## FIL
+
+_FIL_ is the filecoin unit symbol; it is alternatively denoted by the Unicode symbol for an integral with a double stroke (⨎).
+
+## Faucet
+
+A _faucet_ is a service that provides free filecoin. Typically, facuets are run for the benefit of new users in a network, providing them with the necessary seed capital to begin making transactions.
+
+## Fault
+
+When a [storage miner](#storage-miner) fails to complete [Window Proof-of-Spacetime](#window-proof-of-spacetime-windowpost) for a given sector, the Filecoin network registers a _fault_ for that sector, and the miner is [_slashed_](#slash). Storage miners must work to resolve faults quickly, lest they be considered to have abandoned their commitment.
+
+## Filecoin
+
+The term _Filecoin_, in the sense of a proper noun, is used generically to refer to the Filecoin project, network, broader ecosystem and community. As a common noun, the singular form, filecoin, is used to denote amounts of Filecoin currency.
```

Comment:
```suggestion
The term _Filecoin_, in the sense of a proper noun, is used generically to refer to the Filecoin project, network, broader ecosystem, and community. As a common noun, the singular form, Filecoin, is used to denote amounts of Filecoin currency.
```

Reply:
@johnnymatthews  @har00ga 

No problem changing it (indeed, have already accepted a couple of such changes). That said, this contradicts the usual meaning of "common noun", and I thought I'd check one more time just to make sure. For reference, [Bitcoin uses bitcoin](https://bitcoin.org/en/vocabulary#bitcoin)(lowercase) to disambiguate between the project/community and the unit, and it looks like Ethereum [uses ether similarly](https://eth.wiki/en/faqs/glossary). Does the Filecoin project have an official stated preference here?